ajax/libs/rpg-awesome

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rpg-awesome",
+  "filename": "rpg-awesome.min.css",
+  "description": "A fantasy themed font and CSS toolkit.",
+  "keywords": [
+    "rpg",
+    "font",
+    "awesome",
+    "css",
+    "toolkit"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nagoshiashumari/Rpg-Awesome.git"
+  },
+  "authors": [
+    "Daniela Howe",
+    "Ivan Montiel <idmontie@gmail.com>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/nagoshiashumari/Rpg-Awesome",
+  "npmName": "rpg-awesome",
+  "npmFileMap": [{
+    "basePath": "dist",
```

Comment:
There is no `dist` folder in the lib. 
And let's use git auto-update, because there are more versions than npm. (Also the latest version from GitHub is v1.0.0, npm is v0.1.1)
Let's modify the auto-update config as the followings:
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/nagoshiashumari/Rpg-Awesome.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "css/rpg-awesome*",
          "fonts/rpgawesome-webfont*"
        ]
      }
    ]
  }
```

Thanks